draft-ietf-6man-rfc4941bis-latest.xml

```diff
@@ -893,6 +876,17 @@ The default Valid Lifetime for temporary addresses has been reduced from 1 week
       correct prefix and non-existent interface ID). This can be
       addressed by using access control mechanisms on a per-address
       basis on the network egress point.</t>
+
+      <t>XXX: missing security considerations:</t>
+      <t><list style="symbols">
+	<t>The use of a temporary address does not protect the user
+	against tracking in many cases. E.g if the user authenticates
+	to a service or the web service uses cookies or other
+	identifiers in sessions..</t>
```

Comment:
s/[.][.]/./

Reply:
Problem is that if you ever expose an identity using a temporary address, that address is tainted for all other services where you would use it. In the RFC3041 era you might not have had to assume all the servers were exchanging user information. In 2020 you must.